src/components/DeletedNote.tsx

```diff
@@ -20,7 +20,7 @@ const DeletedNote: React.FunctionComponent<DeletedNoteProps> = ({
       <span className="align-middle">Note deleted.</span>
       {onUndelete &&
         <Button color="link" onClick={() => onUndelete(note)} className="ml-1 p-0">
-          undo
+          <u>Undo</u>
```

Comment:
Why is this a u tag?  Seems odd

Reply:
Yeah so we want to add underscore to "Undo" button to make it more obvious